dotnet-engine/Yggdrasil.Benchmarks/Program.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Text.Json;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Running;
+using Newtonsoft.Json.Linq;
+using Yggdrasil;
+
+[Config(typeof(Config))]
+public class YggBench
+{
+    private YggdrasilEngine yggdrasilEngine;
+
+    private class Config : ManualConfig
+    {
+        public Config()
+        {
+            AddColumn(BenchmarkDotNet.Columns.StatisticColumn.OperationsPerSecond);
+        }
+    }
+
+    [GlobalSetup]
+    public void Setup()
+    {
+        var basePath = Path.Combine(
+            "..",
+            "..",
+            "..",
+            "..",
+            "..",
+            "..",
+            "..",
+            "..",
+            "..",
+            "client-specification",
```

Comment:
@daveleek I feel like I've downed a bottle of crazy pills here. Where the hell is this assembly actually executing!?

Surely there's something better than this crazy down path?

Reply:
Long time since I looked at that, but back in the day mstest would move things to run in a subfolder deep down. you could output `TestContext.CurrentContext.TestDirectory` to the console and have a look 😅

Reply:
Don't think that will work, this isn't running in a test context, its a standalone executable so I can remove test harness fluff from the benchmarks

But if nothing stands out to you, it can sit, it works, it's just ugly